### DISCUSSION TITLE
中文翻译错误

### DISCUSSION THREAD

**Discussion by tu6ge** [community/community#66630] [Other Feature Feedback, Questions, & Ideas]
![image](https://github.com/community/community/assets/4115242/b1b59f0f-f392-4928-ae3e-67caedbe4b11)

样板文字应该为“xxx为1个存储库添加了星标”或者“xxx星标了1个存储库”

**Comment by Akash1134** [comment]
Hi @tu6ge , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
